clip2onnx/model_card.py

```diff
@@ -0,0 +1,165 @@
+import inspect
+import logging
+import sys
+import warnings
+from dataclasses import dataclass, field
+from pathlib import Path
+
+import clip
+import open_clip
+import torch
+from PIL import Image
+
+warnings.filterwarnings("ignore", category=UserWarning)
+
+logging.basicConfig(
+    level=logging.INFO, stream=sys.stdout, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+
+
+@dataclass
+class ModelCard:
+    """
+    Open-CLIP Benchmarks
+    > https://github.com/mlfoundations/open_clip/blob/main/docs/openclip_results.csv
+
+    Open-CLIP Model Profile
+    > https://github.com/mlfoundations/open_clip/blob/main/docs/model_profile.csv
+
+    """
+
+    model_name: str
+    """
+    regular model name
```

Comment:
We follow the Google Python style guide. Please move these attribute comments to the docstring at the top of the class, see [relevant style guide part](https://google.github.io/styleguide/pyguide.html#384-classes)